src/client/components/layout/Footer.js

```diff
@@ -18,7 +24,9 @@ function Footer() {
     <footer>
       <AppBar position="static" color="primary" className={classes.appBar}>
         <Toolbar>
-          <Typography>© 2020 Graasp Association</Typography>
+          <Typography variant="subtitle1" className={classes.content}>
+            © 2020 Graasp Association
```

Comment:
@pyphilia, let's use the dynamic code we have for the Graasp Association website.